xbmc/cores/VideoPlayer/VideoPlayerTeletext.h

```diff
@@ -61,7 +61,7 @@ class CDVDTeletextData : public CThread, public IDVDStreamPlayer
   void AllocateCache(int magazine);
 
   int m_speed;
-  TextCacheStruct_t  m_TXTCache;
+  std::shared_ptr<TextCacheStruct_t> m_TXTCache = std::make_shared<TextCacheStruct_t>();
```

Comment:
Just `std::shared_ptr<TextCacheStruct_t> m_TXTCache;` has the same effect

Reply:
No, then m_TXTCache is the empty shared pointer and it crashes in the c'tor.

Reply:
Oh yea, nvm :)